<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>maintenance</class>
 <widget class="QDialog" name="maintenance">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>775</width>
    <height>416</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QStackedWidget" name="maintenance_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>721</width>
     <height>331</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="page">
    <widget class="QGroupBox" name="groupBox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>701</width>
       <height>311</height>
      </rect>
     </property>
     <property name="title">
      <string>Maintenance</string>
     </property>
     <widget class="QTabWidget" name="tabWidget_maintenance">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>661</width>
        <height>271</height>
       </rect>
      </property>
      <property name="currentIndex">
       <number>3</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Ajouter</string>
       </attribute>
       <widget class="QGroupBox" name="groupBox_2">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>631</width>
          <height>221</height>
         </rect>
        </property>
        <property name="title">
         <string>Ajouter Maintenance</string>
        </property>
        <widget class="QLabel" name="label_9">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>30</y>
           <width>131</width>
           <height>20</height>
          </rect>
         </property>
         <property name="text">
          <string>ID de la maintenance:</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_19">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>90</y>
           <width>91</width>
           <height>20</height>
          </rect>
         </property>
         <property name="text">
          <string>ID employé :</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_20">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>140</y>
           <width>91</width>
           <height>20</height>
          </rect>
         </property>
         <property name="text">
          <string>ID équipement :</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_10">
         <property name="geometry">
          <rect>
           <x>350</x>
           <y>20</y>
           <width>91</width>
           <height>20</height>
          </rect>
         </property>
         <property name="text">
          <string>Type du panne :</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_11">
         <property name="geometry">
          <rect>
           <x>340</x>
           <y>80</y>
           <width>151</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>Date de la maintenance :</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="id_maintenance_ajout">
         <property name="geometry">
          <rect>
           <x>140</x>
           <y>30</y>
           <width>113</width>
           <height>20</height>
          </rect>
         </property>
        </widget>
        <widget class="QLineEdit" name="id_employe_ajout">
         <property name="geometry">
          <rect>
           <x>140</x>
           <y>90</y>
           <width>113</width>
           <height>20</height>
          </rect>
         </property>
        </widget>
        <widget class="QLineEdit" name="id_equipememnt_ajout">
         <property name="geometry">
          <rect>
           <x>140</x>
           <y>140</y>
           <width>113</width>
           <height>20</height>
          </rect>
         </property>
        </widget>
        <widget class="QLineEdit" name="type_main_ajout">
         <property name="geometry">
          <rect>
           <x>470</x>
           <y>20</y>
           <width>113</width>
           <height>20</height>
          </rect>
         </property>
        </widget>
        <widget class="QPushButton" name="ajout_maintenance">
         <property name="geometry">
          <rect>
           <x>330</x>
           <y>150</y>
           <width>121</width>
           <height>41</height>
          </rect>
         </property>
         <property name="text">
          <string>Valider</string>
         </property>
        </widget>
        <widget class="QDateEdit" name="dateEdit_mainten_aj">
         <property name="geometry">
          <rect>
           <x>470</x>
           <y>80</y>
           <width>121</width>
           <height>22</height>
          </rect>
         </property>
        </widget>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Afficher</string>
       </attribute>
       <widget class="QTableView" name="tab_maintenance">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>30</y>
          <width>431</width>
          <height>192</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="export_pdf">
        <property name="geometry">
         <rect>
          <x>520</x>
          <y>190</y>
          <width>111</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>Exporter PDF</string>
        </property>
       </widget>
       <widget class="QLabel" name="label">
        <property name="geometry">
         <rect>
          <x>470</x>
          <y>120</y>
          <width>71</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>recherche</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="rechercheav">
        <property name="geometry">
         <rect>
          <x>530</x>
          <y>120</y>
          <width>113</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_3">
       <attribute name="title">
        <string>Modifier</string>
       </attribute>
       <widget class="QGroupBox" name="groupBox_6">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>631</width>
          <height>221</height>
         </rect>
        </property>
        <property name="title">
         <string>Modifier Maintenance</string>
        </property>
        <widget class="QLabel" name="label_15">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>30</y>
           <width>131</width>
           <height>20</height>
          </rect>
         </property>
         <property name="text">
          <string>ID de la maintenance:</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_30">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>90</y>
           <width>91</width>
           <height>20</height>
          </rect>
         </property>
         <property name="text">
          <string>ID employé :</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_31">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>140</y>
           <width>91</width>
           <height>20</height>
          </rect>
         </property>
         <property name="text">
          <string>ID équipement :</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_16">
         <property name="geometry">
          <rect>
           <x>350</x>
           <y>20</y>
           <width>91</width>
           <height>20</height>
          </rect>
         </property>
         <property name="text">
          <string>Type du panne :</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_17">
         <property name="geometry">
          <rect>
           <x>340</x>
           <y>80</y>
           <width>151</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>Date de la maintenance :</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="lineEdit_idmm_2">
         <property name="geometry">
          <rect>
           <x>160</x>
           <y>30</y>
           <width>113</width>
           <height>20</height>
          </rect>
         </property>
        </widget>
        <widget class="QLineEdit" name="lineEdit_idempm_2">
         <property name="geometry">
          <rect>
           <x>160</x>
           <y>90</y>
           <width>113</width>
           <height>20</height>
          </rect>
         </property>
        </widget>
        <widget class="QLineEdit" name="lineEdit_idequipm_2">
         <property name="geometry">
          <rect>
           <x>160</x>
           <y>140</y>
           <width>113</width>
           <height>20</height>
          </rect>
         </property>
        </widget>
        <widget class="QLineEdit" name="lineEdit_typepm_2">
         <property name="geometry">
          <rect>
           <x>480</x>
           <y>20</y>
           <width>113</width>
           <height>20</height>
          </rect>
         </property>
        </widget>
        <widget class="QDateEdit" name="dateEdit_3">
         <property name="geometry">
          <rect>
           <x>480</x>
           <y>80</y>
           <width>110</width>
           <height>22</height>
          </rect>
         </property>
        </widget>
        <widget class="QPushButton" name="edit_mainten">
         <property name="geometry">
          <rect>
           <x>380</x>
           <y>150</y>
           <width>101</width>
           <height>31</height>
          </rect>
         </property>
         <property name="text">
          <string>Modifier</string>
         </property>
        </widget>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_4">
       <attribute name="title">
        <string>Supprimer</string>
       </attribute>
       <widget class="QGroupBox" name="groupBox_7">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>9</y>
          <width>631</width>
          <height>221</height>
         </rect>
        </property>
        <property name="title">
         <string>Supprimer maintenance</string>
        </property>
        <widget class="QLabel" name="label_32">
         <property name="geometry">
          <rect>
           <x>140</x>
           <y>60</y>
           <width>171</width>
           <height>20</height>
          </rect>
         </property>
         <property name="text">
          <string>ID maintenance à supprimer :</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="lineEdit_idm_4">
         <property name="geometry">
          <rect>
           <x>162</x>
           <y>110</y>
           <width>221</width>
           <height>31</height>
          </rect>
         </property>
        </widget>
        <widget class="QPushButton" name="supprimer_ma">
         <property name="geometry">
          <rect>
           <x>230</x>
           <y>150</y>
           <width>91</width>
           <height>31</height>
          </rect>
         </property>
         <property name="text">
          <string>Supprimer</string>
         </property>
        </widget>
       </widget>
      </widget>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="page_2">
    <widget class="QGroupBox" name="groupBox_8">
     <property name="geometry">
      <rect>
       <x>19</x>
       <y>19</y>
       <width>661</width>
       <height>301</height>
      </rect>
     </property>
     <property name="title">
      <string>Equipement</string>
     </property>
     <widget class="QTabWidget" name="tabWidget_equi">
      <property name="geometry">
       <rect>
        <x>6</x>
        <y>29</y>
        <width>621</width>
        <height>261</height>
       </rect>
      </property>
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="tab_9">
       <attribute name="title">
        <string>Ajouter</string>
       </attribute>
       <widget class="QLabel" name="label_4">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>110</y>
          <width>71</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>Nombre :</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_3">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>60</y>
          <width>71</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Type :</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_2">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>20</y>
          <width>71</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>ID :</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_6">
        <property name="geometry">
         <rect>
          <x>280</x>
          <y>60</y>
          <width>71</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Etat :</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_7">
        <property name="geometry">
         <rect>
          <x>280</x>
          <y>110</y>
          <width>71</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Marque :</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_5">
        <property name="geometry">
         <rect>
          <x>280</x>
          <y>20</y>
          <width>71</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Prix :</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_id">
        <property name="geometry">
         <rect>
          <x>70</x>
          <y>20</y>
          <width>113</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_type">
        <property name="geometry">
         <rect>
          <x>70</x>
          <y>60</y>
          <width>113</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_nombre">
        <property name="geometry">
         <rect>
          <x>70</x>
          <y>110</y>
          <width>113</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_prix">
        <property name="geometry">
         <rect>
          <x>340</x>
          <y>20</y>
          <width>113</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_etat">
        <property name="geometry">
         <rect>
          <x>340</x>
          <y>60</y>
          <width>113</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_marque">
        <property name="geometry">
         <rect>
          <x>340</x>
          <y>110</y>
          <width>113</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="ajouter_equipement">
        <property name="geometry">
         <rect>
          <x>490</x>
          <y>70</y>
          <width>75</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Ajouter</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_10">
       <attribute name="title">
        <string>Afficher</string>
       </attribute>
       <widget class="QTableView" name="tab_equipement">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>11</y>
          <width>591</width>
          <height>171</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="pb_trier">
        <property name="geometry">
         <rect>
          <x>360</x>
          <y>190</y>
          <width>91</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Trier</string>
        </property>
       </widget>
       <widget class="QPushButton" name="filtrer">
        <property name="geometry">
         <rect>
          <x>470</x>
          <y>190</y>
          <width>91</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Filtrer</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_11">
       <attribute name="title">
        <string>modifier</string>
       </attribute>
       <widget class="QLabel" name="label_18">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>80</y>
          <width>71</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Type :</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_33">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>25</y>
          <width>71</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>ID :</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_34">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>130</y>
          <width>71</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Nombre :</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_35">
        <property name="geometry">
         <rect>
          <x>294</x>
          <y>80</y>
          <width>71</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>Etat :</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_36">
        <property name="geometry">
         <rect>
          <x>300</x>
          <y>130</y>
          <width>55</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Marque :</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_37">
        <property name="geometry">
         <rect>
          <x>294</x>
          <y>30</y>
          <width>71</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>Prix :</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_idm_2">
        <property name="geometry">
         <rect>
          <x>80</x>
          <y>30</y>
          <width>113</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_typem">
        <property name="geometry">
         <rect>
          <x>80</x>
          <y>80</y>
          <width>113</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_nombrem">
        <property name="geometry">
         <rect>
          <x>80</x>
          <y>130</y>
          <width>113</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_prixm">
        <property name="geometry">
         <rect>
          <x>350</x>
          <y>30</y>
          <width>113</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_etatm">
        <property name="geometry">
         <rect>
          <x>350</x>
          <y>80</y>
          <width>113</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_marquem">
        <property name="geometry">
         <rect>
          <x>360</x>
          <y>130</y>
          <width>113</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_updateequip">
        <property name="geometry">
         <rect>
          <x>240</x>
          <y>182</y>
          <width>91</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>Update</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_12">
       <attribute name="title">
        <string>Supprimer</string>
       </attribute>
       <widget class="QLabel" name="label_38">
        <property name="geometry">
         <rect>
          <x>180</x>
          <y>50</y>
          <width>181</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>ID équipement à supprimer :</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_idsupp">
        <property name="geometry">
         <rect>
          <x>180</x>
          <y>100</y>
          <width>141</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="pb_supprimer">
        <property name="geometry">
         <rect>
          <x>220</x>
          <y>160</y>
          <width>75</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Supprimer</string>
        </property>
       </widget>
      </widget>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QPushButton" name="equipement">
   <property name="geometry">
    <rect>
     <x>144</x>
     <y>362</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Equipement</string>
   </property>
  </widget>
  <widget class="QPushButton" name="maintenance_3">
   <property name="geometry">
    <rect>
     <x>64</x>
     <y>362</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Maintenance</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_play">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>10</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Music play</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_mute">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>10</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Mute</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
